src/slack/slack.js

```diff
@@ -942,6 +942,18 @@ export class Slack {
     ]
   }
 
+  userOrderAttachment(order, fieldsTitle = null, alerts = []) {
+    return {
+      callback_id: order.id,
+      mrkdwn_in: ['fields'],
+      title: 'Order summary',
+      fallback: 'attachment fallback',
```

Comment:
so for `fallback`, the docs are here: https://api.slack.com/reference/messaging/attachments#legacy_fields
<img width="888" alt="image" src="https://user-images.githubusercontent.com/6044119/155704155-71450fa9-7af1-4bd3-8df1-02c7a8535620.png">
so I was right this is used in notifications. let's use `fieldsTitle || 'Please confirm your order:'` here - so a meaningful text is used in notification

BTW: I imagine we will sometime finally get to rewriting the legacy attachment fields to use blocks, but I think that will need to be connected to changing action handling from `callback_id` to `action_id` or `block_id`, so let's try to use the old structure well for now

Reply:
let's use fieldsTitle || 'Please confirm your order:' here - so a meaningful text is used in notification

if we will set 

> text = fieldsTitle || 'Please confirm your order:'
The text will appear between title(Order summary)  and orderFields.

<img width="664" alt="Screen Shot 2022-02-25 at 18 29 07" src="https://user-images.githubusercontent.com/10379289/155723319-0439f425-2624-4370-9b0e-fcdaf9024936.png">

Reply:
I'm not talking about `text` at all. I imagine this:
```js
fallback: fieldsTitle || 'Please confirm your order:'
```